tests/unit/conftest.py

```diff
@@ -76,13 +81,13 @@ async def mattermost_service_mock(
     yield aioresponse_mocker
 
 
-@pytest.fixture()
-async def portainer_service_mock(
```

Comment:
What is the reason that you removed async (turned them into 'regular') for some functions? Is it merely trying to avoid having async (which might be more complicated to maintain) in places where it doesnt give improvements?

Reply:
The general rule is: if a function does not has an ``await`` in in its implementation, it should NOT be marked as ``async``.